### ISSUE TITLE
Subscription Billing Receipt

### ISSUE THREAD

**Issue by the-it-weirdo** [ultralytics/hub#616]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and [discussions](https://github.com/ultralytics/hub/discussions) and found no similar questions.

### Question

Hello Ultralytics Hub,

Could I please get a billing statement for my subscription?

**Comment by sergiuwaxmann** [comment]
Hello @the-it-weirdo!
You can view and download your invoices and receipts in [Settings](https://hub.ultralytics.com/settings?tab=billing) (Billing History card).

![billing_history](https://github.com/ultralytics/hub/assets/47978446/cfe64176-7efa-469b-a87f-9b258e2d0bc0)

**Comment by sergiuwaxmann** [comment]
@the-it-weirdo Can you share the ID of one of your models with me? You can find the model ID in the URL of the model page.

**Comment by the-it-weirdo** [comment]
@sergiuwaxmann Here is the model ID: 1xKPEuwRoyDdwL6uamBI. 

I trained it when I was in the free plan. Since I liked the training process, I got the Pro membership. But I need an invoice for the membership payment.

**Comment by sergiuwaxmann** [comment]
@the-it-weirdo I am sorry, the model ID you shared doesn't belong to you.

**Comment by the-it-weirdo** [comment]
That is odd. @sergiuwaxmann Could you please try this one: txrSdNi43AxTauvmybpb?

**Comment by sergiuwaxmann** [comment]
@the-it-weirdo Surprisingly, we actually had some issues rendering the Billing History card... Thank you for bringing this to our attention! The card should render correctly now.

Can you please check again and let me know if the Billing History card is now visible?

**Comment by the-it-weirdo** [comment]
@sergiuwaxmann Yes it works now. Thank you for the help. I found the invoices I needed....

Also, on a totally unrelated note, and I understand that this might be very unconventional, I wanted to know if you guys are hiring. I am going to finish my Master's degree in Computer Science soon and I am looking for a job.

**Comment by sergiuwaxmann** [comment]
@the-it-weirdo Congratulations! You can see our open positions [here](https://www.ultralytics.com/careers).

**Comment by the-it-weirdo** [comment]
Thank you for all your help. Appreciate it.

**Comment by UltralyticsAssistant** [comment]
@the-it-weirdo you're most welcome! If you have any more questions in the future, feel free to ask. Happy coding! 😊
